### ISSUE TITLE
can this be used to build for ARM arch?

### ISSUE THREAD

**Issue by nmocruz** [spremi/cnijfilter-source-3.80#3]
can this be used to build for ARM arch? I was trying to get a driver for an arm device, but seems the that the best we can do is build this targeting ARM

**Comment by spremi** [comment]
Did you mean, building RPM package?
If so, yes it can be. Build options need to be updated...

**Comment by nmocruz** [comment]
yes, building RPM package, (or other) I don´t know how to do it. there's maybe dependencies to be installed, and things to be changed, but I'm not a linux guy

**Comment by spremi** [comment]
If i get time in near future I will try to do so.
Since you are not a linux guy, do you really need this RPM?

I _ported_ and _fixed_ the driver for personal needs - so far it has been good; but dependencies are getting obsolete.

**Comment by nmocruz** [comment]
I basically want get this driver to install a printer on an arm device

**Comment by spremi** [comment]
Sorry, I had been busy on the work front. Cross compiling for ARM is possible.
Just need to be ported i.e. compiled and packaged. Time and relative priority is important.
Afraid, this one is back in the queue for me on both counts.
